### ISSUE TITLE
Handle handle paywallCost <= 1 cent

### ISSUE THREAD

**Issue by robbyphillips** [kels0la/bittoka#230]
Currently we allow any price to be set for the paywall, but we have a 1 cent 'cut' hard coded into the transaction logic.  We need to either enforce a ~~maximum~~ minimum price for a paywall or determine a different calculation for the Listoka cut.

**Comment by kels0la** [comment]
Minimum price $.02 ($.01 to author, $.01 to platform). I'm content with the flat $.01 fee rather than a percentage-based configuration - at least for now. 

The max is a little trickier. I wanted to say $1.00 at first, but perhaps up to $10.00 would be the way to go.

**Comment by robbyphillips** [comment]
Yeah, I meant minimum of $0.02.  Max price shouldn't matter.  If someone charges too much, they don't get anything.
